azerothcore/azerothcore-wotlk
# fix(DB/Loot): Correct 'Design: Ruby Serpent' loot table

## Changes Proposed:
This item (ID 21949) is supposed to be a world drop. Right now, it's only dropped by a few mobs with a ridiculously low drop rate (see issue 6945 for more details). It's thus in an almost unobtainable state. Making it a world drop would need to be done, but I propose a simpler change, making it at least more Blizzlike than it is as of today:

-  Add the 'Design: Ruby Serpent' (ID 21949) to the loot table of a few creatures with a drop rate high enough so it will at least be obtainable, and low enough so it will still be hard to get.
-  The creatures selected are Elites Dragonkins in Azshara, one of them being already targeted by the RLT 24708.

I'm opened to suggestions concerning the GroupID, I'm not sure if it will be an issue or not for this fix.
## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/1191

## SOURCE:

https://tbc.wowhead.com/item=21949/design-ruby-serpent
https://wowpedia.fandom.com/wiki/Design:_Ruby_Serpent
A lot of the Wowhead drops of this item seem to be lacking. Some of these Elites Dragonkins are dropping it though.

## Tests Performed:

- Ran SQL, no errors/warnings, correct number of rows affected.
- Checked Keira, creature loots were now as expected.


## How to Test the Changes:


1. Check the 4 creatures loot table in Keira to ensure the added item are there and have a 0.5 % drop rate.
2.
3.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
